src/zzz_od/application/hollow_zero/lost_void/operation/lost_void_run_level.py

```
@@ -667,11 +668,33 @@ def move_after_interact(self) -> None:
-            self.ctx.controller.move_s(press=True, press_time=2, release=True)
             if self.interact_target.is_npc:
-                if self.interact_target.name == LostVoidInteractNPC.SCGMDYJY.value:
+                default_move_back: bool = False
+                # 俩npc在一起时, self.interact_target.name 有概率识别错误, 故使用 ao_fei_li_ya_talked 来判断是否选择了关卡武备
+                if self.interact_target.name == LostVoidInteractNPC.AO_FEI_LI_YA.value and not self.ao_fei_li_ya_talked:
+                    self.ao_fei_li_ya_talked = True
+                    # 识别开局右侧是否有两个npc
+                    frame_result: DetectFrameResult = self.ctx.lost_void.detect_to_go(
+                        self.last_screenshot, screenshot_time=self.last_screenshot_time,
+                        ignore_list=self.had_been_list)
+                    with_interact, with_distance, with_entry = self.ctx.lost_void.detector.is_frame_with_all(frame_result)
```

Ruff (0.15.17)

[warning] Unpacked variable `with_distance` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)